minimal-notebook/jupyter_notebook_config.py

```diff
@@ -8,8 +8,6 @@
 PEM_FILE = os.path.join(jupyter_data_dir(), 'notebook.pem')
 
 c = get_config()
-c.NotebookApp.ip = os.getenv('INTERFACE', '') or '*'
```

Comment:
Ooo. Missed this one. Removing this defaults the notebook back to 127.0.0.1. I think we want to retain the * behavior since these are always in docker containers.

I'm guessing that command line opts override a hardcoded '*' here in the config file. But I need to check that to be sure.